src/commands/tileindex-validate/__test__/tileindex.validate.test.ts

```diff
@@ -281,6 +281,20 @@ describe('validatePreset', () => {
     assert.equal(opts['preset'], 'webp');
     assert.ok(opts['reason']?.includes('is not a 8 bits TIFF'));
   });
+  it('should validate multiple tiffs for rgbnir_zstd', async (t) => {
```

Comment:
It would be good to extract the above test into a function to avoid code duplication here.